app/User.php

```diff
@@ -11,13 +11,22 @@ class User extends Authenticatable
 {
     use HasApiTokens, Notifiable;
 
+    
+    /*
+     *
+     *
+     *
+     *
+     */
+    const TYPE = 'admin';
```

Comment:
As I mentioned above, TYPE_ADMIN will be a better name.
What about testees? When you add them there will be also another type.